src/Contracts/Models/TaskObject.cs

```diff
@@ -39,7 +39,7 @@ public class TaskObject
         public TaskDestination[] TaskDestinations { get; set; } = System.Array.Empty<TaskDestination>();
 
         [JsonProperty(PropertyName = "export_destinations")]
-        public TaskDestination[] ExportDestinations { get; set; } = System.Array.Empty<TaskDestination>();
+        public ExportDestination[] ExportDestinations { get; set; } = System.Array.Empty<ExportDestination>();
```

Comment:
should we add using System; in this file to shorten these System.Arrays... ?

Reply:
Removed